{{cookiecutter.repo_name}}/config/settings/production.py

```diff
@@ -118,7 +118,7 @@
 
 # URL that handles the media served from MEDIA_ROOT, used for managing
 # stored files.
-{% if cookiecutter.use_whitenoise == 'y' -%}
+{% if cookiecutter.use_whitenoise == "y" -%}
```

Comment:
The mix of quotes and double-quotes is by me. Accidentally. I process those the same way, hence why my projects mix them. If you want to make the project consistent, then those should remain as single quotes.